rsciio/tests/test_nexus_hdf.py

```diff
@@ -26,9 +26,8 @@
 import traits.api as t
 import h5py
 
-from hyperspy.exceptions import VisibleDeprecationWarning as HSVisibleDeprecationWarning
+from hyperspy.exceptions import VisibleDeprecationWarning as VisibleDeprecationWarning
```

Comment:
```suggestion
from hyperspy.exceptions import VisibleDeprecationWarning
```

Reply:
What's about this?

Reply:
I commited it online, but then overwrote it when pushing the updated version from my PC ;-) Should be fixed now.